docs/source/preferences/PreferencesInEnvisage.rst

```diff
@@ -45,5 +45,8 @@ e.g. To contribute a preference file for my plugin I might use::
       def get_preferences(self, application):
           return ['pkgfile://mypackage:preferences.ini']
 
-.. _PreferencesHelper: ../../enthought/preferences/preferences_helper.py
-.. _ScopedPreferences: ../../enthought/preferences/scoped_preferences.py
+..
+   # substitutions
+
+.. |PreferencesHelper| replace:: :class:`~apptools.preferences.preferences_helper.PreferencesHelper`
+.. |ScopedPreferences| replace:: :class:`~apptools.preferences.scoped_preferences.ScopedPreferences`
```

Comment:
Note that there is an open issue to move "Preferences in Envisage" to `envisage` (#202)
I'm unsure if we want to do that before the upcoming release? (if this were removed/moved over it wouldn't be available until another envisage release happens)

Reply:
It would definitely be nice to copy the docs before the release and like you said, we definitely can't remove the docs from apptools entirely. We'll need to go through a couple of minor versions of apptools before we can finally remove the docs from apptools entirely and simply point to envisage docs.